src/utils/aggregation-utils.js

```diff
@@ -0,0 +1,27 @@
+
+/**
+ * Get an array of object with sorted count and index of bins
+ * @param {Array} bins
+
+ * @return {Array} array of count and index lookup
+ */
+export function getSortedCounts(bins) {
```

Comment:
I wonder if this would be easier to read and work if it was organized as a class with methods?
```
class SortedBins {
   constructor(bins)
   getRangeFromPercentiles([lo, hi]);
}
```